### ISSUE TITLE
[Chat] Enter = Newline on Mobile

### ISSUE THREAD

**Issue by JorenC** [zond/dipact#153]
Not sure if we can have different behaviour on mobile/desktop. 

Ideal:
Desktop. Enter = send, Shift + Enter is newline (also because mouse is further away for a button, shortcut is preffered).
Mobile. Enter = newline. Butotn = send. 

HOWEVER, if this is not possible, then enter could be newline on desktop as well, and ctrl+enter could be send (similar to e.g. outlook).

**Comment by zond** [comment]
The most serious solution I found, http://hgoebl.github.io/mobile-detect.js/, says:

```
Warning
TL;DR: you should not use this library in your HTML page and it’s less reliable when used server-side (Node.js)

As mentioned later, “User-Agent” based detection is not a reliable solution in most cases, because:

The rules (regular expressions) are constantly out-dated and incomplete
You have to update the detection code continuously
There are other ways to detect how your web application should behave:
feature detection, e.g. Modernizr
Media Queries, examples at https://mediaqueri.es/
Maybe there are some libraries out there (which are probably not free) doing a more reliable job
If you still want to (or have to) use this library, you should always encapsulate it with your own code, because chances a very high that you have to tweak the behaviour a bit or are not satisfied with the result of mobile-detect.js. Don’t spread usage of MobileDetect all over your own code! As you can see in the issues, there are some “bugs”, feature-requests, pull-requests where people are not so happy how MobileDetect works. But I cannot change its behaviour from version to version, even if this was reasonable from new users’ point of view. I hope you show understanding.

At least there is a way to monkey-patch the library (see “Extending” below).
```

I don't think this is a good solution. Maybe we should make enter never send?

**Comment by JorenC** [comment]
Can we use ctrl+enter to send on desktop? That does not exist on mobile so it would solve the issue?

**Comment by JorenC** [comment]
I can probably try to figure out how to do it - and we can do it later.
We can also add a small substring in grey under the text input field with "Ctrl+Enter to send", only if the viewport is like larger than 760px - then we have most mobile devices. For tablets, it's not the end of the world that it's being displayed while there is no ctrl button...

**Comment by zond** [comment]
Yes - making enter just do newline, and ctrl + enter send, that's simple. I can add that :D
We can let Ctrl+Enter to send be the "text if empty"-thing in the textarea?

**Comment by JorenC** [comment]
No, don't let it be the "text if empty" thing. That's called a placeholder and it serves a different purpose ;). It also gets removed by typing, which is actually the moment when you need it - you'll start orienting on how to send when you finish typing. 

If you look at the text field here, you can see they added the small text underneath it. If you want, I can add it after you update the control+send thingy, but basically it should be rgba(40,26,26,0.57), M.Typography variant="caption" and aligned with the right side of the input box, underneath it. 

But I can add it as well - I'm just busy with the orders now. And slow goings as usual because I need to debug what all these [Parts] are doing ;)

**Comment by zond** [comment]
I added it as "text if empty" before I read what you wrote. It'll still inform the players, but I'm very happy if you improve it when you have time :)

**Comment by zond** [comment]
Assigned to @JorenC since it's his styling thing left to do.

**Comment by JorenC** [comment]
Fixed in upcoming pull
